core/domain/email_manager.py

```diff
@@ -607,6 +607,23 @@ def send_feedback_message_email(recipient_id, feedback_messages):
         email_subject, email_body, feconf.NOREPLY_EMAIL_ADDRESS)
 
 
+def _can_receive_suggestion_email(user_id, exploration_id):
+    """Returns if user can receive suggestion email
+
+    Args:
+        recipient_id: str. ID of person that should receive the email.
+        exploration_id: str. ID of exploration that received new message.
+
+    Returns:
+        bool. True if user can receive the email, False otherwise."""
+    user_global_prefs = user_services.get_email_preferences(user_id)
+    user_exploration_prefs = (
+        user_services.get_email_preferences_for_exploration(
+            user_id, exploration_id))
+    return (user_global_prefs['can_receive_feedback_message_email']
```

Comment:
I'd suggest creating the new file for the domain objects, so that we follow the other files in core/domain like exp_domain.py and exp_services.py. The *_domain.py files are usually for the domain objects themselves and the *_services.py files are for functions that operate on these objects and other things. So, user_services.py is a bit anomalous at the moment.